.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
```

Comment:
This is the only one I'm not sure on - @miniksa might know more on what encoding we're using. IIRC there are a could files that are encoded very specifically, for tests.

Reply:
Oh, that's probably true, can drop that if requested.

Reply:
This forces all future files to use utf-8 charset? Or do you know if it reformats files opportunistically as they get loaded and resaved?

Reply:
It does both!

Reply:
You could add an override file in the test folder and set `charset = unset` ...

Reply:
I want this, but I also want to then run through every file in the repo and make sure it is UTF-8 and if it isn't or can't be for whatever reason, then we need to make sure it is fixed to handle it.

This means that if we have string constants in it that are characters, we might need to replace them with things like L"\u3482\u2039" or whatnot that cannot be messed up by the encoding.

Since setting the codepage may be fraught with peril, I'd rather you just drop that line from this particular PR so we can have *some* .editorconfig file instead of waiting until we get the perfect one.